webvtt/api/interfaces.html

```diff
@@ -118,6 +118,7 @@
 
 <script type=text/plain id=tested>
 enum AutoKeyword { "auto" };
+typedef (double or AutoKeyword) LineAndPositionSetting;
```

Comment:
IIUC, this "fixes" things because idlharness.js does nothing for typedefs:

> TODO: Test when we actually have something to test this on

So in practice this "hides" these properties from testing. Is there a bug for fixing idlharness.js?